src/main/java/org/testng/internal/objects/SimpleObjectDispenser.java

```diff
@@ -0,0 +1,41 @@
+package org.testng.internal.objects;
+
+import org.testng.TestNGException;
+import org.testng.internal.InstanceCreator;
+import org.testng.internal.objects.pojo.CreationAttributes;
+import org.testng.internal.objects.pojo.DetailedAttributes;
+import org.testng.internal.objects.pojo.BasicAttributes;
+
+/**
+ * A plain vanilla Object dispenser
+ */
+class SimpleObjectDispenser implements IObjectDispenser {
+
+  @Override
+  public void setNextDispenser(IObjectDispenser dispenser) {
+    //We are not going to be doing anything with this downstream dispenser since we are
+    //last in the chain of responsibility.
```

Comment:
An exception could be thrown because it should not happen.

Reply:
Fixed.